src/GoCardlessDropinButton.tsx

```diff
@@ -1,18 +1,18 @@
-import React from "react";
+import React, { FC } from "react";
 import { useGoCardlessDropin } from "./hooks";
 import { GoCardlessDropinOptions } from "./types";
 import "./GoCardlessDropinButton.css";
 
 // Opens a Billing Request Flow in a modal. Relies on GoCardlessDropin being
 // loaded into window, which will only happen if the initialise script has been
 // loaded via a script tag.
-export const GoCardlessDropinButton = (options: GoCardlessDropinOptions) => {
-  const { open } = useGoCardlessDropin({ ...options });
+export const GoCardlessDropinButton: FC<GoCardlessDropinOptions> = (props) => {
```

Comment:
As per GC coding guidelines, We don't use `FC` to define props type. Rather we define it as it was before.

Reply:
All done 👍